scripts/tags_and_languages.py

```diff
@@ -31,6 +31,13 @@
 import requests
 import sys
 import unicodedata
+import argparse
+
+parser = argparse.ArgumentParser()
```

Comment:
You can reuse the help above:
```suggestion
usage, epilog = __doc__.split("\n\n", 1)
parser = argparse.ArgumentParser(usage=usage, epilog=epilog)
```